notebooks/Mitigating Disparities in Ranking from Binary Data.ipynb

```diff
@@ -393,12 +394,12 @@
     "# returns probabilities (of both classes) in predict_proba(), we need to\n",
     "# use the following wrapper for LogisticRegression.\n",
     "\n",
-    "class LogisticRegressionAsRegression:\n",
+    "class LogisticRegressionAsRegression(BaseEstimator, ClassifierMixin):\n",
     "    def __init__(self, logistic_regression_estimator):\n",
     "        self.logistic_regression_estimator = logistic_regression_estimator\n",
     "    \n",
     "    def fit(self, X, y):\n",
-    "        self.logistic_regression_estimator.fit(X, y)\n",
+    "        return self.logistic_regression_estimator.fit(X, y)\n",
     "    \n",
```

Comment:
We should probably return `self` here. However, note that as soon as we introduce `predict_method` parameter as suggested in https://github.com/fairlearn/fairlearn/pull/282/files#r380340267 we will have no need for `LogisticRegressionAsRegression`.

Reply:
Oh wow I love that!

Reply:
Yes, but I've kept that for another PR.